### ISSUE TITLE
[demonoid] Clearance failed after 30 attempt(s). (Config)

### ISSUE THREAD

**Issue by jsorondo** [Jackett/Jackett#8153]
**Please use the search bar** at the top of the page and make sure you are not creating an already submitted issue.
Check closed issues as well, because your issue may have already been fixed.

Please read our [Contributing Guidelines](https://github.com/Jackett/Jackett/blob/master/CONTRIBUTING.md) before submitting your issue to ensure a prompt response to your bug.

### Environment

**OS**:

**.Net Runtime**: [.Net-Core/.Net-Framework/Mono]

**.Net Version**:

**Jackett Version**:

**Last Working Jackett Version**:

**Are you using a proxy or VPN?** [yes/no]

### Description

[List steps to reproduce the error and details on what happens and what you expected to happen]

### Logged Error Messages

[Place any relevant error messages you noticed from the logs here.]

[Make sure you attach the full logs with your personal information removed in case we need more information]

### Screenshots

[Place any screenshots of the issue here if needed]

**Comment by jsorondo** [comment]
I tried adding the cookie value but it sill failed

**Comment by garfield69** [comment]
yes I see this too. I will investigate.

**Comment by garfield69** [comment]
hmmm.
I can only get the cookie to work If I use my browser to first login with my demonoid account.
then the cookie I copy/paste works in the indexer.
If I try to access the site with a cookie from the guest mode demonoid web site page, then the site invokes the cloudflare challenge again despite my having passed that challenge before copying the cookie, and the indexer fails with the 30 attempts error.

**Comment by Jerrk** [comment]
so with currently being unable create a new demonoid account theres no way past this issue?

**Comment by garfield69** [comment]
It's looking that way for now. I have not found a work around so far. :-(

**Comment by Jerrk** [comment]
ah that is unfortunate

**Comment by garfield69** [comment]
The only possibility I can think of is if you use a free VPN that allows you to select different servers in the world (hide.me for example), and you could see in any of them use an IP address  that does not trigger the Demonoid anti-DDoS challenge.
But if the site has turned it on for everybody then this will not work. But worth a check maybe?

**Comment by MalaGaM** [comment]
Need more new HEADERS requests, i think. 
 -H 'Upgrade-Insecure-Requests: 1'
 -H 'TE: Trailers'
User-Agent and cookie

work with curl.

**Comment by RoloSoze** [comment]
@jsorondo Should be fixed with #8324 - Update to version 0.16.105 or later. Feel free to reopen this ticket if updating Jackett does not resolve it.

**Comment by cadatoiva** [comment]
@RoloSoze, this was a cookie issue, not a CloudFlare issue. And appears to still be an issue in the latest version. As mentioned the fix is to login with your account and use the cookie. If you don't have an account, you will be unable to use this tracker.

**Comment by klaypigeon** [comment]
If you have to login, and they are not accepting new accounts, then it should probably be re-categorized as private or semi-private.

**Comment by ngosang** [comment]
It's happening to me too. With the cookie is not enough you have to use the same user-agent in Jackett. @garfield69 it's possible to add a field in the form so the user can copy the user-agent too?

**Comment by garfield69** [comment]
of course we can.
but it does not work.
I have tested this by 
- adding the user-agent to the login method-cookie inputs section
- then accessing demonoid and 
- clicking on its apply-filter button to invoke a search
- solve the challenge and 
- then from the network page, 
- refresh and 
- copy the user-agent 
- and cookie to the config.

but the indexer still returns 0 results and the log shows I still get the _Exception (demonoid): Clearance failed after 30 attempt(s)._ error.

is cardigann ignoring or not replacing the user-agent cookie with the one I provide from the config?
here is a demonoid yml with the user-agent added so you can test and investigate
```yaml
---
  site: demonoid
  name: Demonoid
  description: "Demonoid is a Public torrent site for MOVIES / TV / GENERAL"
  language: en-us
  type: public
  encoding: UTF-8
  followredirect: true
  links:
    - https://www.demonoid.is/
  legacylinks:
    - https://www.dnoid.to/
    - https://demonoid.unblockit.pro/
    - https://demonoid.unblockit.one/ # currently down
    - https://dnoid.black-mirror.xyz/ # proxy's IP blocked
    - https://dnoid.unblocked.casa/ # proxy's IP blocked
    - https://dnoid.proxyportal.fun/ # proxy's IP blocked
    - https://dnoid.uk-unblock.xyz/ # proxy's IP blocked
    - https://dnoid.ind-unblock.xyz/ # proxy's IP blocked

  caps:
    categorymappings:
    # fetch these from https://www.demonoid.is/cached/torrent_categories_script.js?v5
      # Applications
      - {id: 1, cat: PC/Phone-Android, desc: "Applications Android"}
      - {id: 2, cat: PC/Phone-IOS, desc: "Applications Iphone / Ipod touch"}
      - {id: 3, cat: PC, desc: "Applications Linux"}
      - {id: 4, cat: PC/Mac, desc: "Applications Mac"}
      - {id: 5, cat: PC/Phone-Other, desc: "Applications Mobile phone"}
      - {id: 6, cat: PC, desc: "Applications PocketPC"}
      - {id: 7, cat: PC/0day, desc: "Applications Windows"}
      # Audio Books
      - {id: 8, cat: Audio/Audiobook, desc: "Audio Books Action"}
      - {id: 9, cat: Audio/Audiobook, desc: "Audio Books Adventure"}
      - {id: 10, cat: Audio/Audiobook, desc: "Audio Books Biography"}
      - {id: 11, cat: Audio/Audiobook, desc: "Audio Books Childrens"}
      - {id: 12, cat: Audio/Audiobook, desc: "Audio Books Computers and Technology"}
      - {id: 13, cat: Audio/Audiobook, desc: "Audio Books Contemporary"}
      - {id: 14, cat: Audio/Audiobook, desc: "Audio Books Cooking"}
      - {id: 15, cat: Audio/Audiobook, desc: "Audio Books Crafts and Hobbies"}
      - {id: 16, cat: Audio/Audiobook, desc: "Audio Books Educational"}
      - {id: 17, cat: Audio/Audiobook, desc: "Audio Books Fantasy"}
      - {id: 18, cat: Audio/Audiobook, desc: "Audio Books Fiction"}
      - {id: 19, cat: Audio/Audiobook, desc: "Audio Books General"}
      - {id: 20, cat: Audio/Audiobook, desc: "Audio Books History"}
      - {id: 21, cat: Audio/Audiobook, desc: "Audio Books Horror"}
      - {id: 22, cat: Audio/Audiobook, desc: "Audio Books Humor"}
      - {id: 23, cat: Audio/Audiobook, desc: "Audio Books Literary"}
      - {id: 24, cat: Audio/Audiobook, desc: "Audio Books Magazine"}
      - {id: 25, cat: Audio/Audiobook, desc: "Audio Books Mainstream"}
      - {id: 26, cat: Audio/Audiobook, desc: "Audio Books Medicine and Health"}
      - {id: 27, cat: Audio/Audiobook, desc: "Audio Books Mystery and Suspense"}
      - {id: 28, cat: Audio/Audiobook, desc: "Audio Books Newspaper"}
      - {id: 29, cat: Audio/Audiobook, desc: "Audio Books Nonfiction"}
      - {id: 30, cat: Audio/Audiobook, desc: "Audio Books Other"}
      - {id: 31, cat: Audio/Audiobook, desc: "Audio Books Paranormal"}
      - {id: 32, cat: Audio/Audiobook, desc: "Audio Books Religion"}
      - {id: 33, cat: Audio/Audiobook, desc: "Audio Books Romance"}
      - {id: 34, cat: Audio/Audiobook, desc: "Audio Books RPG"}
      - {id: 35, cat: Audio/Audiobook, desc: "Audio Books Sci-Fi"}
      - {id: 36, cat: Audio/Audiobook, desc: "Audio Books Self-help"}
      - {id: 37, cat: Audio/Audiobook, desc: "Audio Books Suspense"}
      - {id: 38, cat: Audio/Audiobook, desc: "Audio Books Textbook"}
      - {id: 39, cat: Audio/Audiobook, desc: "Audio Books Thriller"}
      - {id: 40, cat: Audio/Audiobook, desc: "Audio Books Western"}
      - {id: 41, cat: Audio/Audiobook, desc: "Audio Books Young Adult"}
      # Books
      - {id: 42, cat: Books/Ebook, desc: "Books Action and Adventure"}
      - {id: 43, cat: Books/Ebook, desc: "Books Biography"}
      - {id: 44, cat: Books/Ebook, desc: "Books Childrens"}
      - {id: 45, cat: Books/Ebook, desc: "Books Computers and Technology"}
      - {id: 46, cat: Books/Ebook, desc: "Books Contemporary"}
      - {id: 47, cat: Books/Ebook, desc: "Books Cooking"}
      - {id: 48, cat: Books/Ebook, desc: "Books Crafts and Hobbies"}
      - {id: 49, cat: Books/Ebook, desc: "Books Educational"}
      - {id: 50, cat: Books/Ebook, desc: "Books Fantasy"}
      - {id: 51, cat: Books/Ebook, desc: "Books Fiction"}
      - {id: 52, cat: Books/Ebook, desc: "Books General"}
      - {id: 53, cat: Books/Ebook, desc: "Books History"}
      - {id: 54, cat: Books/Ebook, desc: "Books Horror"}
      - {id: 55, cat: Books/Ebook, desc: "Books Humor"}
      - {id: 56, cat: Books/Ebook, desc: "Books Literary"}
      - {id: 57, cat: Books/Magazines, desc: "Books Magazine"}
      - {id: 58, cat: Books/Ebook, desc: "Books Mainstream"}
      - {id: 59, cat: Books/Ebook, desc: "Books Medicine and Health"}
      - {id: 60, cat: Books/Ebook, desc: "Books Mystery and Suspense"}
      - {id: 61, cat: Books/Ebook, desc: "Books Newspaper"}
      - {id: 62, cat: Books/Ebook, desc: "Books Nonfiction"}
      - {id: 63, cat: Books/Ebook, desc: "Books Other"}
      - {id: 64, cat: Books/Ebook, desc: "Books Paranormal"}
      - {id: 65, cat: Books/Ebook, desc: "Books Religion"}
      - {id: 66, cat: Books/Ebook, desc: "Books Romance"}
      - {id: 67, cat: Books/Ebook, desc: "Books RPG"}
      - {id: 68, cat: Books/Ebook, desc: "Books Sci-Fi"}
      - {id: 69, cat: Books/Ebook, desc: "Books Self-help"}
      - {id: 70, cat: Books/Ebook, desc: "Books Suspense"}
      - {id: 71, cat: Books/Ebook, desc: "Books Textbook"}
      - {id: 72, cat: Books/Ebook, desc: "Books Thriller"}
      - {id: 73, cat: Books/Ebook, desc: "Books Western"}
      - {id: 74, cat: Books/Ebook, desc: "Books Young Adult"}
      # Comics
      - {id: 75, cat: Books/Comics, desc: "Comics Action / Adventure"}
      - {id: 76, cat: Books/Comics, desc: "Comics Crime"}
      - {id: 77, cat: Books/Comics, desc: "Comics Drama"}
      - {id: 78, cat: Books/Comics, desc: "Comics Fantasy"}
      - {id: 79, cat: Books/Comics, desc: "Comics Historical fiction"}
      - {id: 80, cat: Books/Comics, desc: "Comics Horror"}
      - {id: 81, cat: Books/Comics, desc: "Comics Illustrated novel"}
      - {id: 82, cat: Books/Comics, desc: "Comics Manga"}
      - {id: 83, cat: Books/Comics, desc: "Comics Other"}
      - {id: 84, cat: Books/Comics, desc: "Comics Real-Life"}
      - {id: 85, cat: Books/Comics, desc: "Comics Sci-Fi"}
      - {id: 86, cat: Books/Comics, desc: "Comics Super Hero"}
      # Games
      - {id: 87, cat: PC, desc: "Games DOS"}
      - {id: 88, cat: Console/Other, desc: "Games Dreamcast"}
      - {id: 89, cat: Console/Other, desc: "Games Emulators"}
      - {id: 90, cat: Console/Other, desc: "Games GameBoy"}
      - {id: 91, cat: Console/Other, desc: "Games GameCube"}
      - {id: 92, cat: PC, desc: "Games Linux"}
      - {id: 93, cat: PC/Mac, desc: "Games Mac"}
      - {id: 94, cat: PC/Phone-Other, desc: "Games Mobile phone"}
      - {id: 95, cat: Console/NDS, desc: "Games Nintendo DS"}
      - {id: 96, cat: Console/Other, desc: "Games Palm"}
      - {id: 97, cat: Console/PS3, desc: "Games Playstation"}
      - {id: 98, cat: Console/Other, desc: "Games PocketPC"}
      - {id: 99, cat: Console/PSP, desc: "Games PSP"}
      - {id: 100, cat: Console/Wii, desc: "Games Wii / Wii U"}
      - {id: 101, cat: PC/Games, desc: "Games Windows"}
      - {id: 102, cat: Console/Xbox, desc: "Games XBox"}
      # Anime
      - {id: 158, cat: TV/Anime, desc: "Anime Sci-Fi"}
      - {id: 103, cat: TV/Anime, desc: "Anime Action"}
      - {id: 104, cat: TV/Anime, desc: "Anime Adventure"}
      - {id: 105, cat: TV/Anime, desc: "Anime Comedy"}
      - {id: 106, cat: TV/Anime, desc: "Anime Drama"}
      - {id: 107, cat: TV/Anime, desc: "Anime Fantasy"}
      - {id: 108, cat: TV/Anime, desc: "Anime Horror"}
      - {id: 109, cat: TV/Anime, desc: "Anime Other"}
      - {id: 157, cat: TV/Anime, desc: "Anime Romance"}
      # Movies
      - {id: 151, cat: Movies, desc: "Movies Action"}
      - {id: 110, cat: Movies, desc: "Movies Adventure"}
      - {id: 236, cat: Movies, desc: "Movies Animation"}
      - {id: 111, cat: Movies, desc: "Movies Biography"}
      - {id: 152, cat: Movies, desc: "Movies Comedy"}
      - {id: 112, cat: Movies, desc: "Movies Concerts"}
      - {id: 113, cat: Movies, desc: "Movies Crime"}
      - {id: 114, cat: Movies, desc: "Movies Documentary"}
      - {id: 115, cat: Movies, desc: "Movies Drama"}
      - {id: 116, cat: Movies, desc: "Movies Family"}
      - {id: 153, cat: Movies, desc: "Movies Fantasy"}
      - {id: 117, cat: Movies, desc: "Movies Horror"}
      - {id: 118, cat: Movies, desc: "Movies Musical"}
      - {id: 119, cat: Movies, desc: "Movies Mystery"}
      - {id: 120, cat: Movies, desc: "Movies Other"}
      - {id: 121, cat: Movies, desc: "Movies Romance"}
      - {id: 154, cat: Movies, desc: "Movies Sci-Fi"}
      - {id: 159, cat: Movies, desc: "Movies Short-Film"}
      - {id: 160, cat: Movies, desc: "Movies Sports"}
      - {id: 161, cat: Movies, desc: "Movies Thriller"}
      - {id: 162, cat: Movies, desc: "Movies Trailers"}
      - {id: 163, cat: Movies, desc: "Movies War"}
      - {id: 164, cat: Movies, desc: "Movies Western"}
      # Music
      - {id: 122, cat: Audio, desc: "Music Alternative"}
      - {id: 123, cat: Audio, desc: "Music Bluegrass"}
      - {id: 124, cat: Audio, desc: "Music Blues"}
      - {id: 125, cat: Audio, desc: "Music Childrens"}
      - {id: 126, cat: Audio, desc: "Music Christian"}
      - {id: 127, cat: Audio, desc: "Music Classical"}
      - {id: 128, cat: Audio, desc: "Music Comedy"}
      - {id: 129, cat: Audio, desc: "Music Contemporary African"}
      - {id: 130, cat: Audio, desc: "Music Country"}
      - {id: 131, cat: Audio, desc: "Music Dance / Disco"}
      - {id: 132, cat: Audio, desc: "Music Drum and Bass"}
      - {id: 133, cat: Audio, desc: "Music Electro / Techno"}
      - {id: 134, cat: Audio, desc: "Music Folk"}
      - {id: 165, cat: Audio, desc: "Music Gospel"}
      - {id: 166, cat: Audio, desc: "Music Grunge"}
      - {id: 167, cat: Audio, desc: "Music Hip-Hop / Rap"}
      - {id: 168, cat: Audio, desc: "Music Indie"}
      - {id: 169, cat: Audio, desc: "Music Industrial"}
      - {id: 170, cat: Audio, desc: "Music J-Pop"}
      - {id: 171, cat: Audio, desc: "Music Jazz"}
      - {id: 172, cat: Audio, desc: "Music Latin American"}
      - {id: 173, cat: Audio, desc: "Music Melodic"}
      - {id: 174, cat: Audio, desc: "Music Metal"}
      - {id: 175, cat: Audio, desc: "Music Other"}
      - {id: 176, cat: Audio, desc: "Music Pop"}
      - {id: 177, cat: Audio, desc: "Music Punk"}
      - {id: 178, cat: Audio, desc: "Music Radio Show"}
      - {id: 179, cat: Audio, desc: "Music Reggae"}
      - {id: 180, cat: Audio, desc: "Music Rhythm and blues"}
      - {id: 181, cat: Audio, desc: "Music Rock"}
      - {id: 182, cat: Audio, desc: "Music Soul"}
      - {id: 183, cat: Audio, desc: "Music Soundtrack"}
      - {id: 184, cat: Audio, desc: "Music Trance"}
      - {id: 185, cat: Audio, desc: "Music Trip Hop"}
      # Music Videos
      - {id: 135, cat: Audio/Video, desc: "Music Videos Alternative"}
      - {id: 136, cat: Audio/Video, desc: "Music Videos Bluegrass"}
      - {id: 137, cat: Audio/Video, desc: "Music Videos Blues"}
      - {id: 138, cat: Audio/Video, desc: "Music Videos Childrens"}
      - {id: 139, cat: Audio/Video, desc: "Music Videos Christian"}
      - {id: 186, cat: Audio/Video, desc: "Music Videos Classical"}
      - {id: 187, cat: Audio/Video, desc: "Music Videos Comedy"}
      - {id: 188, cat: Audio/Video, desc: "Music Videos Contemporary African"}
      - {id: 189, cat: Audio/Video, desc: "Music Videos Country"}
      - {id: 190, cat: Audio/Video, desc: "Music Videos Dance / Disco"}
      - {id: 191, cat: Audio/Video, desc: "Music Videos Drum and Bass"}
      - {id: 192, cat: Audio/Video, desc: "Music Videos Electro / Techno"}
      - {id: 193, cat: Audio/Video, desc: "Music Videos Folk"}
      - {id: 194, cat: Audio/Video, desc: "Music Videos Gospel"}
      - {id: 195, cat: Audio/Video, desc: "Music Videos Grunge"}
      - {id: 196, cat: Audio/Video, desc: "Music Videos Hip-Hop / Rap"}
      - {id: 197, cat: Audio/Video, desc: "Music Videos Indie"}
      - {id: 198, cat: Audio/Video, desc: "Music Videos Industrial"}
      - {id: 199, cat: Audio/Video, desc: "Music Videos J-Pop"}
      - {id: 200, cat: Audio/Video, desc: "Music Videos Jazz"}
      - {id: 201, cat: Audio/Video, desc: "Music Videos Latin American"}
      - {id: 202, cat: Audio/Video, desc: "Music Videos Melodic"}
      - {id: 203, cat: Audio/Video, desc: "Music Videos Metal"}
      - {id: 204, cat: Audio/Video, desc: "Music Videos Other"}
      - {id: 205, cat: Audio/Video, desc: "Music Videos Pop"}
      - {id: 206, cat: Audio/Video, desc: "Music Videos Punk"}
      - {id: 207, cat: Audio/Video, desc: "Music Videos Radio Show"}
      - {id: 208, cat: Audio/Video, desc: "Music Videos Reggae"}
      - {id: 209, cat: Audio/Video, desc: "Music Videos Rhythm and blues"}
      - {id: 210, cat: Audio/Video, desc: "Music Videos Rock"}
      - {id: 211, cat: Audio/Video, desc: "Music Videos Soul"}
      - {id: 212, cat: Audio/Video, desc: "Music Videos Soundtrack"}
      - {id: 213, cat: Audio/Video, desc: "Music Videos Trance"}
      - {id: 214, cat: Audio/Video, desc: "Music Videos Trip Hop"}
      # Pictures
      - {id: 140, cat: Other, desc: "Pictures Art"}
      - {id: 141, cat: Other, desc: "Pictures Commercial"}
      - {id: 142, cat: Other, desc: "Pictures Glamour"}
      - {id: 215, cat: Other, desc: "Pictures Other"}
      - {id: 216, cat: Other, desc: "Pictures Photojournalism"}
      - {id: 217, cat: Other, desc: "Pictures Snapshots"}
      - {id: 218, cat: Other, desc: "Pictures Sports"}
      - {id: 219, cat: Other, desc: "Pictures Wildlife"}
      # TV
      - {id: 155, cat: TV, desc: "TV Action"}
      - {id: 143, cat: TV, desc: "TV Adventure"}
      - {id: 144, cat: TV, desc: "TV Biography"}
      - {id: 145, cat: TV, desc: "TV Cartoons"}
      - {id: 146, cat: TV, desc: "TV Comedy"}
      - {id: 147, cat: TV, desc: "TV Concerts"}
      - {id: 148, cat: TV, desc: "TV Crime"}
      - {id: 149, cat: TV, desc: "TV Documentary"}
      - {id: 156, cat: TV, desc: "TV Drama"}
      - {id: 220, cat: TV, desc: "TV Family"}
      - {id: 221, cat: TV, desc: "TV Fantasy"}
      - {id: 222, cat: TV, desc: "TV Horror"}
      - {id: 223, cat: TV, desc: "TV Musical"}
      - {id: 224, cat: TV, desc: "TV Mystery"}
      - {id: 225, cat: TV, desc: "TV Other"}
      - {id: 226, cat: TV, desc: "TV Reality"}
      - {id: 227, cat: TV, desc: "TV Romance"}
      - {id: 228, cat: TV, desc: "TV Sci-Fi"}
      - {id: 229, cat: TV, desc: "TV Short film"}
      - {id: 230, cat: TV, desc: "TV Sports"}
      - {id: 231, cat: TV, desc: "TV Talk show"}
      - {id: 232, cat: TV, desc: "TV Thriller"}
      - {id: 233, cat: TV, desc: "TV Trailers"}
      - {id: 234, cat: TV, desc: "TV War"}
      - {id: 235, cat: TV, desc: "TV Western"}
      # cat 7 misc has no subcats

    modes:
      search: [q, imdbid]
      tv-search: [q, season, ep, imdbid]
      movie-search: [q, imdbid]
      music-search: [q]

  settings:
    - name: cookie
      type: text
      label: Cookie
    - name: info
      type: info
      label: How to get the Cookie
      default: "<ol><li>Access this tracker with your browser<li>click on the <b>Apply Filter</b> button on the page to invoke the search and solve the challenge<li>Open the <b>DevTools</b> panel by pressing <b>F12</b><li>Select the <b>Network</b> tab<li>Click on the <b>Doc</b> button<li>Refresh the page by pressing <b>F5</b><li>Select the <b>Headers</b> tab<li>Find <b>'cookie:'</b> in the <b>Request Headers</b> section<li><b>Select</b> and <b>Copy</b> the whole cookie string <i>(everything after 'cookie: ')</i> and <b>Paste</b> here.</ol>"
    - name: useragent
      type: text
      label: User-Agent
      label: User-Agent
    - name: info_useragent
      type: info
      label: How to get the User-Agent
      default: "<ol><li>From the same place you fetched the cookie,<li>Find <b>'user-agent:'</b> in the <b>Request Headers</b> section<li><b>Select</b> and <b>Copy</b> the whole user-agent string <i>(everything after 'user-agent: ')</i> and <b>Paste</b> here.</ol>"
    - name: sort
      type: select
      label: Sort requested from site
      default: "added"
      options:
        "added": "created"
        "seeders": "seeders"
        "size": "size"
    - name: type
      type: select
      label: Order requested from site
      default: "desc"
      options:
        "desc": "desc"
        "asc": "asc"

  login:
    method: cookie
    inputs:
      cookie: "{{ .Config.cookie }}"
      user-agent: "[ .Config.useragent ]"

  download:
    selector: a[href^="magnet:?xt="]
    attribute: href

  search:
    paths:
      # https://www.demonoid.is/files/?c154&language=0&quality=0&seeded=2&query=world&to=1&sort=
      # https://www.demonoid.is/files/?seeded=2&language=0&quality=0&to=1&query=world
      - path: files/
    inputs:
      $raw: "{{ range .Categories }}c{{.}}&{{end}}"
      # 0 seeded 1 unseeded 2 both
      seeded: 2
      # 0 all 1 English etc
      language: 0
      # 0 all
      quality: 0
      # search in titles only
      to: "{{ if .Query.IMDBID }}{{else}}1{{end}}"
      query: "{{ if .Query.IMDBID }}{{ .Query.IMDBID }}{{else}}{{ .Keywords }}{{end}}"
      sort: "{{ .Config.sort }}"
      order: "{{ .Config.type }}"
      # does not return imdb link in results

    rows:
      selector: table.font_12px tr:has(td[class^="tone_1"])
      after: 1
      dateheaders:
        selector: tr td.added_today:not(:contains("Sponsored links"))
        filters:
          - name: replace
            args: ["Added ", ""]
          - name: replace
            args: ["on ", ""]

    fields:
      title:
        selector: a[href^="/files/details/"]
      title:
        selector: a[href^="/files/details/"]
        optional: true
        attribute: title
      category:
        selector:  span > a[href*="&subcategory="]
        attribute: href
        optional: true
        filters:
          - name: querystring
            args: subcategory
      details:
        selector: a[href^="/files/details/"]
        attribute: href
      download:
        selector: a[href^="/files/details/"]
        attribute: href
      description:
        selector: td:nth-last-child(9)
      size:
        selector: td:nth-last-child(6)
      grabs:
        selector: td:nth-last-child(4)
      seeders:
        selector: td:nth-last-child(3)
      leechers:
        selector: td:nth-last-child(2)
      downloadvolumefactor:
        text: 0
      uploadvolumefactor:
        text: 1
# engine n/a
```

**Comment by ilike2burnthing** [comment]
![demonoid](https://user-images.githubusercontent.com/59480337/81958252-9eb6bb80-9605-11ea-9008-927164445fb3.png)

Was too good to last...

**Comment by SPDurkee** [comment]
Here's the entry in my log file pertaining to this issue...

CloudflareSolverRe.Exceptions.CloudflareClearanceException: Clearance failed after 10 attempt(s).
  at CloudflareSolverRe.ClearanceHandler.SendAsync (System.Net.Http.HttpRequestMessage request, System.Threading.CancellationToken cancellationToken) [0x001f9] in <26b83cb0a9ac472388afda7d5e026324>:0 
  at System.Net.Http.HttpClient.SendAsyncWorker (System.Net.Http.HttpRequestMessage request, System.Net.Http.HttpCompletionOption completionOption, System.Threading.CancellationToken cancellationToken) [0x000ca] in <791ca8fbb0c442e984539829ef259511>:0 
  at Jackett.Common.Utils.Clients.HttpWebClient.Run (Jackett.Common.Utils.Clients.WebRequest webRequest) [0x0048a] in <ecf315ca6b3349eb9ee595f5db8136a9>:0 
  at Jackett.Common.Utils.Clients.WebClient.GetString (Jackett.Common.Utils.Clients.WebRequest request) [0x0010b] in <ecf315ca6b3349eb9ee595f5db8136a9>:0 
  at Jackett.Common.Indexers.BaseWebIndexer.RequestStringWithCookies (System.String url, System.String cookieOverride, System.String referer, System.Collections.Generic.Dictionary`2[TKey,TValue] headers) [0x000cb] in <ecf315ca6b3349eb9ee595f5db8136a9>:0 
  at Jackett.Common.Indexers.CardigannIndexer.TestLogin () [0x000b9] in <ecf315ca6b3349eb9ee595f5db8136a9>:0 
  at Jackett.Common.Indexers.CardigannIndexer.ApplyConfiguration (Newtonsoft.Json.Linq.JToken configJson) [0x000d2] in <ecf315ca6b3349eb9ee595f5db8136a9>:0 
  at Jackett.Server.Controllers.IndexerApiController.UpdateConfig (Jackett.Common.Models.DTO.ConfigItem[] config) [0x0012d] in <a2725418de634d30aaa2c5374fe808de>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor+TaskOfIActionResultExecutor.Execute (Microsoft.AspNetCore.Mvc.Infrastructure.IActionResultTypeMapper mapper, Microsoft.Extensions.Internal.ObjectMethodExecutor executor, System.Object controller, System.Object[] arguments) [0x00071] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync () [0x00131] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync () [0x0009e] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow (Microsoft.AspNetCore.Mvc.Filters.ActionExecutedContext context) [0x0001b] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next (Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker+State& next, Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker+Scope& scope, System.Object& state, System.Boolean& isCompleted) [0x00382] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync () [0x0002f] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter () [0x0009f] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow (Microsoft.AspNetCore.Mvc.Filters.ResourceExecutedContext context) [0x0001b] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next (Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker+State& next, Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker+Scope& scope, System.Object& state, System.Boolean& isCompleted) [0x00840] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync () [0x0002f] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync () [0x0012e] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke (Microsoft.AspNetCore.Http.HttpContext httpContext) [0x001cb] in <6092a16d93814eba828b517a2b132f80>:0 
  at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke (Microsoft.AspNetCore.Http.HttpContext context) [0x00384] in <427697fe42b7459ba5302fb76d339d3b>:0 
  at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke (Microsoft.AspNetCore.Http.HttpContext context) [0x0043e] in <f352e566abf6421e87eafbcf57a0b237>:0 
  at Jackett.Server.Middleware.CustomExceptionHandler.Invoke (Microsoft.AspNetCore.Http.HttpContext httpContext) [0x0008a] in <a2725418de634d30aaa2c5374fe808de>:0

**Comment by garfield69** [comment]
this is pending PR #8431 which is on hold for now.

**Comment by ngosang** [comment]
They don't use CloudFlare anymore.
